packages/runtime/src/prop-controllers/index.ts

```diff
@@ -65,3 +69,4 @@ export type {
 export { RichTextPropControllerMessageType, TableFormFieldsMessageType } from './instances'
 export * as Introspection from './introspection'
 export { copy as imageCopy } from './copy/image'
+export {copy as imagesCopy} from './copy/images'
```

Comment:
Looks like there's some spacing inconsistencies here—Prettier should automatically fix this

Reply:
Done